### DISCUSSION TITLE
No Weather entry on Settings

### DISCUSSION THREAD

**Discussion by BaliB4ba** [valinet/ExplorerPatcher#3767] [Q&A] [answered]
Hi all.

Just installed latest EP 22621.3880.66.6 on a fresh Win11  24H2, and there does not appear to be a Weather entry in the settings.

Has been it been taken off?
Cheers.
Bali.
![2024-10-04_13-38](https://github.com/user-attachments/assets/fe994530-3735-4364-af5b-35a7f50784fc)

**Comment by pyrates999** [comment, marked as answer]
weather is not yet supported on the windows 10 (ExplorerPatcher) taskbar: https://github.com/valinet/ExplorerPatcher/issues/3546

**Reply by BaliB4ba** [reply]
Thank you, Sir!
